config/feature_flags.yml

```
@@ -1,4 +1,14 @@
 feature_flags:
+  enable_login_with_azure:
```

```suggestion
  sign_in_with_active_directory:
```

We don't need `enable` as the flag itself is enabling something :)